src/entity.rs

```diff
@@ -184,6 +186,22 @@ pub struct ClaimValue {
     pub references: Vec<ReferenceGroup>,
 }
 
+
+/// A site name, as used in the sitelinks.
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+pub struct SiteName(pub String);
+
+/// A sitelink value.
+#[derive(Debug, Clone, PartialEq, Default, Serialize, Deserialize)]
+pub struct SitelinkValue {
+    /// The title of the site link.
+    pub title: String,
+    /// The badges of the article.
+    pub badges: Vec<WikiId>,
```

Comment:
This should be a `Vec<Qid>`, since badges are always items (and not lexemes or properties).

Reply:
Okay, I changed it! Using this lib to get into the details of wikidata, good that you already know more of the details :-)